examples/image_retraining/retrain.py

```diff
@@ -964,10 +964,32 @@ def export_model(module_spec, class_count, saved_model_dir):
     )
 
 
+def logging_level_verbosity(logging_verbosity):
+    """Converts logging_level into TensorFlow logging verbosity value
+
+      Args:
+        logging_level: String value representing logging level: 'DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL'
+      """
+    nameToLevel = {
+        'FATAL': tf.logging.FATAL,
+        'ERROR': tf.logging.ERROR,
+        'WARN': tf.logging.WARN,
+        'INFO': tf.logging.INFO,
+        'DEBUG': tf.logging.DEBUG
+    }
+
+    if logging_verbosity in nameToLevel:
+        return nameToLevel[logging_verbosity]
+
+    return None
```

Comment:
I think bad values should raise an error. That saves you the if-guard below, and catches bad input values.

Reply:
It's already caught by args parser, but added Exception also here ^